lte/gateway/c/session_manager/test/test_local_enforcer.cpp

```diff
@@ -429,28 +429,216 @@ TEST_F(LocalEnforcerTest, test_multi_version_reporting) {
   insert_static_rule(1, "", "rule1");
   RuleRecordTable table;
   auto record_list = table.mutable_records();
-  // ipv4 usage
   create_rule_record(
       IMSI1, test_cfg_.common_context.ue_ipv4(), "rule1", 1, 10, 20, 0, 0,
       record_list->Add());
-  // ipv6 usage for the same charging key and subscriber
   create_rule_record(
-      IMSI1, test_cfg_.common_context.ue_ipv6(), "rule1", 1, 25, 35, 0, 0,
+      IMSI1, test_cfg_.common_context.ue_ipv4(), "rule1", 1, 25, 35, 0, 0,
       record_list->Add());
   create_rule_record(
-      IMSI1, test_cfg_.common_context.ue_ipv6(), "rule1", 2, 5, 105, 0, 0,
+      IMSI1, test_cfg_.common_context.ue_ipv4(), "rule1", 2, 5, 105, 0, 0,
       record_list->Add());
 
   auto update = SessionStore::get_default_session_update(session_map);
   auto uc     = get_default_update_criteria();
+  // update to version 3
   session_map[IMSI1][0]->increment_rule_stats("rule1", &uc);
-
   session_map[IMSI1][0]->increment_rule_stats("rule1", &uc);
-  // update to version2
   session_map[IMSI1][0]->increment_rule_stats("rule1", &uc);
   local_enforcer->aggregate_records(session_map, table, update);
   assert_charging_credit(session_map, IMSI1, SESSION_ID_1, USED_RX, {{1, 30}});
   assert_charging_credit(session_map, IMSI1, SESSION_ID_1, USED_TX, {{1, 140}});
+  EXPECT_EQ(
+      session_map[IMSI1][0]->get_policy_stats("rule1").current_version, 3);
+}
+
+TEST_F(LocalEnforcerTest, test_old_version_reporting) {
+  /*
+  Tests reporting of tx and rx values of old versions, after
```

Comment:
could you fix the style to this

```
/*
 * Tests reporting of tx and rx values of old versions, after
 * updating to a new version, as well as whether rule records of old
 * versions are disregarded appropriately
 */

Reply:
Hey @veshkemburu sorry to nitpick but do you mind using `//` for the comments in test files? (Should prob document this somewhere)

It's often really useful to be able to comment out chunks of tests when debugging so I prefer to have it all be not blocked.

```
// Tests reporting of tx and rx values of old versions, after
// updating to a new version, as well as whether rule records of old
// versions are disregarded appropriately
```